codex_of_the_damned/templates/archetypes/bibliodeque/akunanse-classic.html

```diff
@@ -0,0 +1,34 @@
+{% extends "archetypes/_layout.html" %}
+{% import "archetypes/_layout-erratum.html" as erratum with context %}
+{% import "archetypes/bibliodeque/akunanse-classic.json" as deck_json %}
+
+{% block meta %}
+<meta property="decklist" content='{{ deck_json | tojson }}' />
+{% endblock %}
+
+{% block archetype_name %}
+<h1><img src="{{ url_for('static', filename='img/icons/clan-akunanse.svg') }}"> Akunanse Classic</h1>
+{% endblock %}
+
+{% block archetype %}
+{% trans trimmed
+
+%}
+<h2>Highlights</h2>
+<p>
+
+</p>
+<h2>Tips & Tricks</h2>
+<p>
+
+</p>
+<h2>Variants</h2>
+<p>
```

Comment:
Not sure we'll use this variant section for decks of the bibliodèque

Reply:
Fair enough

Reply:
resolved